README.md

```diff
@@ -635,6 +635,10 @@ Returns an object composed from key-value pairs.
 
   fromPairs([['a', 1], ['b', 2]]);
   // => { 'a': 1, 'b': 2 }
+
+  // Native(ES2019)
+  Object.fromEntries([['a', 1], ['b', 2]])
```

Comment:
I'm not sure if we should add this. Object.fromEntries are not supported by Edge and IE

Reply:
actually, Edge starts to support it from 79.... https://caniuse.com/#search=fromEntries

@Trick93 could you update the version support so that we can merge this?

Reply:
thks, I have updated the version support